concepts/admission-webhook-cert-management/README.md

```diff
@@ -0,0 +1,48 @@
+# Admission webhook certificate management
+
+This document proposes several ways to manage TLS certificates for custom admission webhooks. 
+
+## Problem
+
+Kubernetes admission controllers act as gatekeepers intercepting API requests and can change the request object or deny its entry to the cluster.
+It is possible to extend the compiled-in admission controllers with custom webhooks using the [dynamic admission control mechanism](https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/).
+
+<img src="assets/admission-controller-certs.drawio.svg">
+
+A handful of Kyma operators are deployed with custom validating admission webhooks (api-gateway, pod-preset, serverless, telemetry, etc.). They use different apporaches to manage certificates, which can be categorized into 2 groups.
+
+1. Use Helm built-in crypto functions to generate certificates (used by pod-preset, telemetry):
```

Comment:
The two options are alternatives to each other, not subsequent steps; so I'd use a bullet list, not a numbered list.

```suggestion
* Use Helm built-in crypto functions to generate certificates (used by pod-preset, Telemetry):
```

Reply:
Converted it to a subheader